ircb/identd.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+import asyncio
+import logging
+import logging.config
+import traceback
+
+import ircb.stores
+
+from ircb.storeclient import NetworkStore, initialize as storeclient_initialize
+from ircb.config import settings
+
+logger = logging.getLogger('identd')
+
+
+class IdentdServerProtocol(asyncio.Protocol):
+    def connection_made(self, transport):
+        print('New client connection received')
+        self.transport = transport
+
+    def data_received(self, data):
+        asyncio.Task(self.handle_data_received(data))
+
+    @asyncio.coroutine
+    def handle_data_received(self, data):
+        print(data)
```

Comment:
Yes, these print statements will be removed as I will be doing some minor changes the way we log.